generic-sensor/generic-sensor-tests.js

```diff
@@ -141,11 +175,15 @@ function runGenericSensorTests(sensorType) {
     }
   }, "no exception is thrown when calling stop() on already stopped sensor");
 
-  async_test(t => {
-    window.onmessage = t.step_func(e => {
-      assert_equals(e.data, "SecurityError");
-      t.done();
-    });
+  promise_test(t => {
+    return load_iframe(make_script(sensorType))
+      .then(iframe => {
+        iframe.contentWindow.postMessage({}, '*');
```

Comment:
@alexshalamov, seems there's a bug in ```submisssions``` wpt server that it references the ```/generic-sensor/generic-sensor-tests.js``` from https://w3c-test.org/generic-sensor/generic-sensor-tests.js rather than  https://w3c-test.org/submissions/6185/generic-sensor/generic-sensor-tests.js.
It works fine on my personal repo, https://honry.github.io/demo/accelerometer/Accelerometer.https.html